### DISCUSSION TITLE
혹시 오픈채팅 현재 공지 id 값을 받아오는 방법 아시나요?

### DISCUSSION THREAD

**Discussion by ha1fstack** [storycraft/node-kakao#189] [Q&A]
.

**Comment by samknp** [comment]
v4는 아직 board기능이 없는 상태인거 같고
v3는 requestPostList 하셨을때 가장 처음에  뜨는게 가장 최신의 post입니다.
그 중에서 공지 찾으시려면 해당 object의 object_type이 'text'인지 찾으시면 되셔요.

**Reply by raravel** [reply]
`v3.1.2` 에는 create_at 이라고 포스트 생성 날짜가 있었는데, 이게  postList 의 `date`로 바뀐건지 최신 v3 소스에서는 볼 수가 없네요. `data` 값이 `Date.now()` 로 생성된 날짜값 인것 같습니다. 이거 기준으로 정렬해 보세요.

**Comment by storycraft** [comment]
채팅방 공지 같은 메타는 변경시 CHGMETA 패킷이 오게되는데 이것만으로는 공지의 id 값을 가져 올수 없습니다. 채팅방 게시판은 별개의 api를 사용하는데 메타 변경시 CHGMETA 패킷 뿐만 아니라 CHGMOMETAS라는 패킷도 오게되며 이 안에 해당 서비스 에서 사용하는 값들이 포함되어 있습니다. 다만 현재 node-kakao에서는 MOMETA 관련 패킷은 처리하고 있지 않습니다.
